build_tools/azure/install.sh

```diff
@@ -120,6 +135,20 @@ python_environment_install() {
         pip install https://github.com/joblib/joblib/archive/master.zip
         echo "Installing pillow master"
         pip install https://github.com/python-pillow/Pillow/archive/main.zip
+    elif [[ "$DISTRIB" == "pip-nogil" ]]; then
+        setup_ccache  # speed-up the build of CPython it-self
+        ORIGINAL_FOLDER=`pwd`
+        cd ..
+        git clone --depth 1 https://github.com/colesbury/nogil
+        cd nogil
+        ./configure && make -j 2
+        ./python -m venv $ORIGINAL_FOLDER/$VIRTUALENV
+        cd $ORIGINAL_FOLDER
+        source $VIRTUALENV/bin/activate
+
+        python -m pip install -U pip
+        echo "Installing build dependepencies with pip from the nogil repository"
```

Comment:
Can we leave a comment about how `nogil`'s pip uses a different index: https://d1yxz45j0ypngg.cloudfront.net ?

Reply:
Done.